lib/serum/dev_server.ex

```diff
@@ -16,7 +16,10 @@ defmodule Serum.DevServer do
     import Supervisor.Spec
 
     uniq = Base.url_encode64(:crypto.strong_rand_bytes(6))
-    site = "/tmp/serum_" <> uniq
+
+    site =
+      "serum_" <> uniq
+      |> Path.expand(System.tmp_dir!())
```

Comment:
I usually don't use a pipe if there is only one function to pipe the value through.
Just this one line looks nicer to me:

```elixir
site = Path.expand("serum_" <> uniq, System.tmp_dir!())
```

Reply:
I think the latter looks better. I'll take your suggestion. Thank you for providing me with a good project!